dopamine/discrete_domains/run_experiment/run_async_training.py

```diff
@@ -0,0 +1,21 @@
+# coding=utf-8
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Runs asynchronous training."""
+
+from dopamine.discrete_domains.run_experiment import run_experiment
```

Comment:
I think there are two different things going on in this CL.  One of them is adding this class, the other is refactoring `run_experiment` functionality into different files.  I don't like the refactor for what it does to `run_experiment`.  But maybe it's necessary.  It's hard for me as reviewer to tell, at this point, or to suggest alternatives.

I think either I'd like to see the new functionality in a set of one or more CLs and then a follow-up CL with the refactor **or** a chat about the motivation for this change.

Reply:
As discussed, the refactoring is a requirement to avoid circular imports.

Reply:
moved async runner to same file as other runners.